tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/typedescriptors/TypeReferenceTSymbolTest.java

```diff
@@ -112,6 +112,41 @@ public void testRecordField() {
         assertEquals(((TypeReferenceTypeSymbol) (recordFieldSymbol).typeDescriptor()).definition(), typeSymbol.get());
         assertEquals(((recordFieldSymbol).typeDescriptor()).typeKind(), TypeDescKind.TYPE_REFERENCE);
         assertEquals(((recordFieldSymbol).typeDescriptor()).getName().get().toString(), "Age");
+    }
+
+    @Test
+    public void testReferringATypeRef1() {
+        Optional<Symbol> symbol = model.symbol(srcFile, from(46, 8));
+        TypeSymbol type = ((VariableSymbol) symbol.get()).typeDescriptor();
```

Comment:
Missing `assertTrue(symbol.isPresent())` check.

Reply:
This is not an issue as this is just a util symbol used and not the subject being tested. That behaviour is covered under other tests for `symbol()`.